src/databricks/labs/dqx/pii/pii_detection_funcs.py

```diff
@@ -0,0 +1,153 @@
+import logging
+import json
+import re
+import warnings
+
+from collections.abc import Callable
+
+import pyspark.sql.connect.session
+from presidio_analyzer import AnalyzerEngine
+from presidio_analyzer.nlp_engine import NlpEngineProvider
+
+from pyspark.sql import Column
+from pyspark.sql.functions import concat_ws, lit, pandas_udf, PandasUDFType
+
+from databricks.labs.dqx.rule import register_rule
+from databricks.labs.dqx.check_funcs import make_condition, _get_normalized_column_and_expr
+from databricks.labs.dqx.pii.nlp_engine_config import NLPEngineConfig
+
+logging.getLogger("presidio_analyzer").setLevel(logging.ERROR)
+logger = logging.getLogger(__name__)
+
+_default_nlp_engine_config = NLPEngineConfig.SPACY_SMALL
+
+
+@register_rule("row")
+def contains_pii(
```

Comment:
the logic is reverted. The quality check should fail if the condition is violated. The function name should be `not_contains_pii`. 
For example: `is_not_null` check will fail if the value is null.